### PR TITLE
Bugfix/rlau 1071 token symbol conflicts

### PR DESCRIPTION
**Pull request by talekhinezh**

### CODE REVIEW

File: radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/TokenDefinitionsReducer.java [hunk 1]
```diff
@@ -58,17 +57,30 @@ public TokenDefinitionsState reduce(TokenDefinitionsState state, TransitionedPar
...
+						: mintedOrBurnedAmount
+				);
 
-			TokenDefinitionReference tokenDefinitionReference = p instanceof MintedTokensParticle
-				? ((MintedTokensParticle) p).getTokenDefinitionReference() : ((BurnedTokensParticle) p).getTokenDefinitionReference();
-			return state.mergeSupplyChange(tokenDefinitionReference, change);
+				TokenDefinitionReference tokenDefinitionReference = p instanceof MintedTokensParticle
+					? ((MintedTokensParticle) p).getTokenDefinitionReference() : ((BurnedTokensParticle) p).getTokenDefinitionReference();
+				return state.mergeSupplyChange(tokenDefinitionReference, change);
+			} else if (t.getTransition() == ParticleTransition.U2N) {
```
Comment (mpsandiford) [inline radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/TokenDefinitionsReducer.java]: It seems to me that these two are really the same thing, but with different signs.  In an ideal world, they would be the same code.  For me this isn't a deal breaker, but maybe could be tidied up.

Comment (flotothemoon) [inline radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/TokenDefinitionsReducer.java]: This logic could be simplified a bit by merging the two branches and negating the logic?

Comment (flotothemoon) [reply radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/TokenDefinitionsReducer.java]: Ah yes, you were faster, ha. Deleting my comment..

Comment (talekhinezh) [reply radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/TokenDefinitionsReducer.java]: I won't worry about cleaning this up as I think experience here shows that this rewindability of state should probably be handled at a lower layer.

**Pull request merged**